notebooks/api/0.8/11-container-images-k8s.ipynb

```diff
@@ -493,8 +493,7 @@
     "assert workerimage is not None, str([image.__dict__ for image in image_list])\n",
     "assert workerimage.is_built is not None, str(workerimage)\n",
     "assert workerimage.built_at is not None, str(workerimage)\n",
-    "assert workerimage.image_hash is not None, str(workerimage)\n",
-    "assert image_list[workerimage.built_image_tag] == workerimage"
```

Comment:
Q: Is this assert statement not needed anymore?

Reply:
Nope not required. Earlier we could index the worker image list with the tag name. But that implementation is buggy,  when you have different images with the same tag. So now we return all images but let go of indexing them via tag.